generators/init/index.js

```diff
@@ -0,0 +1,16 @@
+/* eslint no-underscore-dangle: 0 */
+
+/*
+* License-Identifier: Apache-2.0
+*/
+const Generator = require('yeoman-generator');
+const simpleFabricaConfigJson = require('../../samples/fabricaConfig-1org-1channel-1chaincode');
```

Comment:
nie sądzisz, że defaultowy to powinien być ten z raftem?

Reply:
imo powinno być solo. 
- typowo jeśli uruchamia się minialne środowisko to robi się to na solo.
- raft oznacza że musi być włączony TLS, a tego możemy nie chcieć przy bardzo prostym projekcie.

Reply:
do przegadania

Reply:
zamykam - zostaje prosta sieć